src/include/sof/schedule/task.h

```diff
@@ -43,6 +43,7 @@ enum task_state {
 
 struct task {
 	uint64_t start;
+	uint64_t period;
```

Comment:
Complexity is starting to grow here, can you comment all these fields now to help other understand the logic.

Reply:
Done.